webui/src/app/components/appdetail/appdetail.component.html

```diff
@@ -35,7 +35,7 @@ <h5 class="oui-header_5">
       <div>
         <hr>
         <h5 class="oui-header_5">
-          <i class="oui-icon oui-icon-page_line" aria-hidden="true"></i>
+          <app-help-widget key="help-what-is-a-repository"></app-help-widget><i class="oui-icon oui-icon-page_line" aria-hidden="true"></i>
           {{ 'REPOSITORY' | translate }}
         </h5>
```

Comment:
The icon should appear *after* the string `Repository`.

Reply:
Done